skills/meta/writing-skills/SKILL.md

```
@@ -71,16 +80,31 @@ API docs, syntax guides, tool documentation (office docs)
+```
+${SUPERPOWERS_SKILLS_ROOT}/
+  skills/
+    category/
+      skill-name/
+        SKILL.md              # Main reference (required)
+        supporting-file.*     # Only if needed
+```
+
+**Project skills** (version-controlled with project):
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)